### DISCUSSION TITLE
Disable secure connection via main.conf

### DISCUSSION THREAD

**Discussion by dev-enphase** [bluez/bluez#720] [Q&A]
Hi team,

I am using Bluez 5.66 and trying to disable secure connection as I want to use NoInputNoOutput capability. I disabled the secure connection setting in main.conf but still in btmon logs I can see secure connection. This setting have no effect on secure connection 
Noinputnooutput not working for me when device is paired second time. Any more settings I need to do in main.conf for NoInputNoOutput capability.?
I am using c++ Dbus for handling pairing request

**Comment by GandhiKeyur8** [comment]
Hi,

You can use below in main.conf file. 

+# Specify the policy to the JUST-WORKS repairing initiated by peer
+# Possible values: "never", "confirm", "always"
+# Defaults to "never"
+#**JustWorksRepairing = always**
